plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/TypeHelper.java

```diff
@@ -84,7 +84,7 @@ public static org.apache.kudu.Type toKuduClientType(Type type)
             return org.apache.kudu.Type.DECIMAL;
         }
         if (type == DateType.DATE) {
-            return org.apache.kudu.Type.STRING;
+            return org.apache.kudu.Type.DATE;
```

Comment:
Older Kudu versions don't support DATE type, right? What's the EOL of those versions? 
We should use the different type mapping based on the server version if this change breaks existing environments.

Reply:
The trino kudu connector has dependency of kudu version 1.15.0 whitch supporting date. End users do not understand why trino kudu connector does not support  what  their server support  with  kudu client 1.15.0.
I don't have EOL for kudu

Reply:
The internal dependency is unrelated to users. Our official documentation mentions 1.13.0 is the minimum supported. I would recommend making sure the EOL.

Reply:
I think Kudu supports date from 1.12.0 https://kudu.apache.org/docs/prior_release_notes.html#rn_1.12.0 so I guess we are good here. WDYT @ebyhr ?

Reply:
👍